tools/flaky-test-reporter/github_issue.go

```diff
@@ -488,8 +494,8 @@ func (gi *GithubIssue) processGithubIssueForRepo(rd *RepoData, flakyIssuesMap ma
 
 // analyze all results, figure out flaky tests and processing existing auto:flaky issues
 func (gi *GithubIssue) processGithubIssues(repoDataAll []*RepoData, dryrun bool) error {
-	messagesMap := make(map[string][]string)
-	errMap := make(map[string][]error)
+	messagesMap := make(map[string]map[string][]string)
```

Comment:
I just feel like this needs to be defined in a struct. Its difficult to keep track of which map we are using and what it relates to :thinking:

Reply:
This would be good, but @chaodaiG has ideas for a larger change that would make this outdated anyways. I can add a comment to clarify for the time being.

Reply:
That's right, I totally agree with this idea. It would be even nicer to move this logic out of this file and make a shared components that can be used by both github and slack